commons/src/main/java/org/frankframework/util/EnumUtils.java

```diff
@@ -120,4 +121,22 @@ private static <E extends Enum<E>> E parseIgnoreCase(final Class<E> enumClass, f
 		}
 		return null;
 	}
+
+	/**
+	 * Check if the Enum value is deprecated.
+	 * @param value Enum value
+	 * @return true if @Deprecated annotation is present on the Enum value
+	 */
+	public static boolean isEnumDeprecated(Enum<?> value) {
+		if (value == null) {
+			return false;
+		}
+		try {
+			Field field = value.getClass().getField(value.name());
+			Deprecated annotation = field.getAnnotation(Deprecated.class);
+			return annotation != null;
+		} catch (NoSuchFieldException | SecurityException ignored) {
+		}
+		return false;
```

Comment:
You could move the `return false` into the `catch {}`, so that it's not empty?

Reply:
Ah yes, done.